src/main.py

```diff
@@ -39,16 +39,28 @@ def main() -> None:
         log.warning("Stopping CoinTaxman.")
         return
 
+    # Merge identical operations together, which makes it easier to match
+    # buy/sell to get prices from csv, match fees and reduces database access
+    # (as long as there are only one buy/sell pair per time,
+    # might be problematic otherwhise).
+    book.merge_identical_operations()
+    # Resolve dependencies between withdrawals and deposits, which is
+    # necessary to correctly fetch prices and to calculate p/l.
+    book.resolve_deposits()
     book.get_price_from_csv()
+    book.match_fees_with_operations()
+
     taxman.evaluate_taxation()
-    evaluation_file_path = taxman.export_evaluation_as_csv()
+    evaluation_file_path = taxman.export_evaluation_as_excel()
     taxman.print_evaluation()
 
     # Save log
     log_file_path = evaluation_file_path.with_suffix(".log")
```

Comment:
This does not work if there is already a .log file with that name, but no .xlsx file. Shouldn't be a critical issue, but I just noticed it (I have older .log/.csv exports from working on the main branch, but no .xlsx files with the same name).

Reply:
Next file name finder is now checking against multiple extensions (xlsx and log) which solves this problem.